packages/babel-plugin-transform-computed-properties/test/fixtures/spec/mixed/actual.js

```diff
@@ -1,6 +1,11 @@
+var heh = "heh";
+var noo = "noo";
+var foo = "foo";
+
 var obj = {
```

Comment:
I don't understand how it worked before, either `foo` and `bar` were undefined in the score or you now re-defined them?

Reply:
This was purely a fixture/snapshot test, I've added also `exec.js` and that's why I had to add those to the scope in actual/expected fixtures (like you have noticed - they were previously undefined).

Reply:
Ok that make sense